frontend/dockerfile/dockerfile_test.go

```diff
@@ -5479,6 +5480,40 @@ RUN echo $(hostname) | grep foo
 	}
 }
 
+func testEmptyStages(t *testing.T, sb integration.Sandbox) {
+	integration.SkipOnPlatform(t, "windows")
```

Comment:
Why?

Reply:
Just a default atm. There is a separate task to enable integration tests one-by-one in wcow. Not specific to the feature.